conbench/tests/entities/test_commit.py

```diff
@@ -161,10 +181,11 @@ def test_backfill_default_branch_commits():
     )
 
     # this should backfill all 335 default-branch commits before that one
-    backfilled_commits = backfill_default_branch_commits(repository, commit_1)
```

Comment:
The changes to this test are because `backfill_default_branch_commits()` no longer returns anything, because we can't tell which commits were inserted and which were ignored during the upsert transaction.

Reply:
> because we can't tell which commits were inserted and which were ignored during the upsert transaction.

For the record: if we ever need this, then I believe the sqlalchemy docs show how to get a result from this single transaction, showing which rows were newly inserted.

Reply:
I briefly looked in the docs yesterday, but couldn't quite find an example of what I wanted, especially for `on_conflict_do_nothing()` (as opposed to `on_conflict_do_update()`). However I'm sure it's possible to use `.returning()` in some way that it returns only the commits that were inserted. We'd just have to look into it a little more.